ci/cron/src/Main.hs

```diff
@@ -317,20 +318,32 @@ download_assets :: FilePath -> GitHubRelease -> IO ()
 download_assets tmp release = do
     manager <- HTTP.newManager TLS.tlsManagerSettings
     tokens <- Control.Concurrent.QSem.newQSem 20
-    Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) (\url ->
+    Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) $ \url ->
```

Comment:
More `$` more happiness.

Reply:
@garyverhaegen-da is not going to be happy , sorry Gary :)

Reply:
I don't think I am either, but I don't think I can really judge Haskell code properly. 😂

Reply:
I don’t feel too strongly about it so if you both prefer parentheses here I’m happy to go back. Personally, I find it quite hard to track parentheses beyond a few lines without an editor that does it for me so I tend to opt for `$` for larger blocks and parentheses for small stuff.

Reply:
I was mostly joking at my own lack of experience with Haskell. 😉

Reply:
Why would you ever use an editor that does not track parentheses? 😕

I really don't mind using `$`; I even use it myself sometimes.

Reply:
Reading git diffs unfortunately doesn’t track parentheses for me :slightly_smiling_face: